v4/go.mod

```diff
@@ -4,6 +4,7 @@ go 1.12
 
 require (
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
```

Comment:
we don't need to keep the original module reference, do we???

Reply:
also, if you changed go.mod to use a completely different package (i.e. form3tech-oss/jwt-go), then how did you avoid having to make source changes where the jwt-go package is being used?

Reply:
Fixed.